lore/estimators/keras.py

```diff
@@ -177,63 +177,99 @@ def build_inputs(self):
             if hasattr(encoder, 'sequence_length'):
                 for i in range(encoder.sequence_length):
                     inputs[encoder.sequence_name(i)] = Input(shape=(1,), name=encoder.sequence_name(i))
+                    if encoder.twin:
+                        inputs[encoder.sequence_name(i, suffix='_twin')] = Input(shape=(1,), name=encoder.sequence_name(i, suffix='_twin'))
             else:
                 inputs[encoder.name] = Input(shape=(1,), name=encoder.name)
+                if encoder.twin:
+                    inputs[encoder.twin_name] = Input(shape=(1,), name=encoder.twin_name)
         return inputs
 
     @timed(logging.INFO)
     def build_embedding_layer(self, inputs, tower):
         embeddings = {}
+
         for encoder in self.model.pipeline.encoders:
             embed_name = str(tower) + '_embed_' + encoder.name
+            embed_name_twin = str(tower) + '_embed_' + encoder.name + '_twin'
             embed_size = encoder.embed_scale * self.embed_size
-
-            if isinstance(encoder, Continuous):
-                embedding = Dense(embed_size, activation='relu', name=embed_name)
+            if isinstance(encoder, Pass):
```

Comment:
Why not have an embedding for Pass? What about reserving that behavior strictly for `encoder.embed_layers == 0`?

Reply:
I was think Pass as an encoder which doesn't need any embedding and we want to pass the input directly.